### PR TITLE
support for exporting build env parameters for local dev work

### PR DESCRIPTION
**Pull request by dgraeber**
*Issue #, if available:*
#151 

*Description of changes:*
Added support to export the environment parameters of an existing build run based on build id....to support ease of local development .

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: seedfarmer/commands/_parameter_commands.py [hunk 1]
```diff
@@ -36,6 +36,13 @@ def generate_export_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[L
     return envs
 
 
+def generate_export_raw_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[List[str]]:
+    envs: List[str] = []
+    if metadata is not None and metadata.keys() is not None:
```
Comment (chamcca) [inline seedfarmer/commands/_parameter_commands.py:41]: `metadata.keys()` will never return `None` even if the metadata dict is empty..

File: seedfarmer/commands/_parameter_commands.py [hunk 2]
```diff
@@ -36,6 +36,13 @@ def generate_export_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[L
...
+        envs = [f"export {upper_snake_case(k)}={metadata[k]}" for k in metadata.keys()]
```
Comment (chamcca) [inline seedfarmer/commands/_parameter_commands.py:42]: this appears to result in an export statement like:
```
export SOME_VARIABLE=some_value
```
but if the value is JSON (which is common) or the value has spaces in it, then the export won't work correctly. should probably wrap the value in single quotes like:
```
export SOME_VARIABLE='some value`
```

Comment (dgraeber) [reply seedfarmer/commands/_parameter_commands.py]: Added and corrected

File: seedfarmer/commands/_parameter_commands.py (hunk 1 above)
Comment (dgraeber) [reply seedfarmer/commands/_parameter_commands.py]: removed

**Review by chamcca — APPROVED** [review]

**Pull request merged**